packages/@react-spectrum/searchwithin/src/SearchWithin.tsx

```diff
@@ -13,25 +13,43 @@
 import {classNames, SlotProvider, useFocusableRef, useResizeObserver, useStyleProps} from '@react-spectrum/utils';
 import {Field} from '@react-spectrum/label';
 import {FocusableRef} from '@react-types/shared';
+// @ts-ignore
+import intlMessages from '../intl/*.json';
 import React, {useCallback, useLayoutEffect, useRef, useState} from 'react';
 import {SpectrumSearchWithinProps} from '@react-types/searchwithin';
 import styles from '@adobe/spectrum-css-temp/components/searchwithin/vars.css';
+import {useFormProps} from '@react-spectrum/form';
+import {useId, useLabels} from '@react-aria/utils';
 import {useLabel} from '@react-aria/label';
+import {useMessageFormatter} from '@react-aria/i18n';
 import {useProvider, useProviderProps} from '@react-spectrum/provider';
 
 function SearchWithin(props: SpectrumSearchWithinProps, ref: FocusableRef<HTMLElement>) {
   props = useProviderProps(props);
+  props = useFormProps(props);
+  let formatMessage = useMessageFormatter(intlMessages);
   let {styleProps} = useStyleProps(props);
-  let {labelProps, fieldProps} = useLabel(props);
   let {
     children,
     isDisabled,
     isRequired,
-    label,
-    'aria-label': ariaLabel,
-    'aria-labelledby': ariaLabelledby
+    label
   } = props;
 
+  let defaultAriaLabel = formatMessage('search');
+  // Get label and group props (aka fieldProps)
+  let {labelProps, fieldProps} = useLabel(props);
```

Comment:
we'll need to have some way to suppress the console warning from useLabel, that was why we'd reversed the order, so now we're getting that console warning again
given that we have a default aria label, lets add it to the props object we pass to useLabel if there isn't already a label or aria-label on the props object, is that enough?

Reply:
I think that should do it. Something like this should work:
```jsx
  if (!props['aria-label'] && !props['aria-labelledby']) {
    props['aria-label'] = defaultAriaLabel;
  }
```
I'll push a fix.

Reply:
Ah I see, good call @snowystinger. In that case I think we can do away with the `useLabels` call below entirely right? Will have to replace instances of `groupProps` with `fieldProps`

Reply:
@LFDanLu good catch, just removed